docs/Contributing/Audit-logs.md

```diff
@@ -542,6 +542,24 @@ This activity contains the following fields:
 }
 ```
 
+## fleet_enrolled
+
+Generated when a host is enrolled to Fleet (Fleet's agent fleetd is installed).
+
+This activity contains the following fields:
+- "host_serial": Serial number of the host.
```

Comment:
Hey @marko-lisica, why is the serial not displayed in the example below?

Reply:
@pintomi1989 Good catch, thanks. I forgot to update example, but it's supposed to show serial number and host display name.